src/bots/HighSellVolumeBot.js

```diff
@@ -0,0 +1,255 @@
+const loggerNamespace = 'dx-service:bots:HighSellVolumeBot'
+const AuctionLogger = require('../helpers/AuctionLogger')
+const Bot = require('./Bot')
+const Logger = require('../helpers/Logger')
+
+const logger = new Logger(loggerNamespace)
+const auctionLogger = new AuctionLogger(loggerNamespace)
+
+const numberUtil = require('../helpers/numberUtil')
+const formatUtil = require('../helpers/formatUtil')
+
+const ENSURE_LIQUIDITY_PERIODIC_CHECK_MILLISECONDS =
+  process.env.BUY_LIQUIDITY_BOT_CHECK_TIME_MS || (60 * 1000) // 1 min
+
+class HighSellVolumeBot extends Bot {
+  constructor ({
+    name,
+    // eventBus,
+    // liquidityService,
+    dxInfoService,
+    marketService,
+    botAddress,
+    markets,
+    slackClient,
+    botTransactionsSlackChannel,
+    // buyLiquidityRules,
+    notifications,
+    checkTimeInMilliseconds = ENSURE_LIQUIDITY_PERIODIC_CHECK_MILLISECONDS
+  }) {
+    super(name)
+
+    // this._eventBus = eventBus
+    // this._liquidityService = liquidityService
+    this._dxInfoService = dxInfoService
+    this._marketService = marketService
+    this._botAddress = botAddress
+    this._markets = markets
+    this._slackClient = slackClient
+    this._botTransactionsSlackChannel = botTransactionsSlackChannel
+    // this._buyLiquidityRules = buyLiquidityRules
+    this._notifications = notifications
+    this._checkTimeInMilliseconds = checkTimeInMilliseconds
+
+    this._lastCheck = null
+    this._lastBuy = null
+    this._lastError = null
+  }
+
+  async _doStart () {
+    logger.debug({ msg: 'Initialized bot: ' + this.name })
+
+    this._markets.forEach(market => {
+      const sellToken = market.tokenA
+      const buyToken = market.tokenB
+      this._doRoutineLiquidityCheck(sellToken, buyToken)
+    })
+
+    // Check the liquidity periodically
+    setInterval(() => {
+      this._markets.forEach(market => {
+        const sellToken = market.tokenA
+        const buyToken = market.tokenB
+        this._doRoutineLiquidityCheck(sellToken, buyToken)
+      })
+    }, this._checkTimeInMilliseconds)
+  }
+
+  async _doStop () {
+    logger.debug({ msg: 'Bot stopped: ' + this.name })
+  }
+
+  async _doRoutineLiquidityCheck (sellToken, buyToken) {
+    // Do ensure liquidity on the market
```

Comment:
This comment is a C&P :)

Reply:
You're right, using same structure as buy bot

I'll fix it